apps/reader/workers/error-worker/classify_error.ts

```diff
@@ -0,0 +1,40 @@
+export enum PostgresErrors {
```

Comment:
Where are these error coders from? 
does drizzle define them, 
can they be found with a string search?

Reply:
The codes are defined by postgres